src/components/date.js

```diff
@@ -32,7 +32,7 @@ export default class FormattedDate extends Component {
     }
 
     render() {
-        const {formatDate}      = this.context.intl;
+        const {formatDate, textComponent} = this.context.intl;
```

Comment:
Ah ok. I thought we were talking about a type here. but you are talking about creating an alias